chebpy/core/chebtech.py

```diff
@@ -79,7 +80,7 @@ def initvalues(cls, values, *, interval=None):
     def __init__(self, coeffs, interval=None):
         interval = interval if interval is not None else prefs.domain
         self._coeffs = np.array(coeffs, dtype=float)
-        self._interval = np.array(interval)
+        self._interval = Interval(*interval)
```

Comment:
Ideally, this would be the only place to convert to `Interval`, but we require `interval.hscale` inside `initfun_adaptive` too.